app/src/main/java/org/oppia/android/app/player/state/DragDropSortInteractionView.kt

```diff
@@ -76,6 +75,7 @@ class DragDropSortInteractionView @JvmOverloads constructor(
     //  this more efficiently and cleanly than always relying on notifying of potential changes in the adapter when the
     //  type is set (plus the type ought to be permanent).
     this.isMultipleItemsInSamePositionAllowed = isAllowed
+    singleTypeBuilderFactory = BindableAdapter.SingleTypeBuilder.Factory(Fragment())
```

Comment:
Reminder to move this back to injection.

Reply:
Hi @BenHenning moving this back to injection does not work i believe it's due to the mentioned data race happening in the view as mentioned on comments by the previous codeowner, had mentioned this on my updates earlier. Please check and advise i reverted SelectionInteractionView to not use DI to provide BindableAdapter since due to the mentioned data race happening on attached is called very late.